bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -91,7 +91,10 @@
 			<channel id="locked" typeId="locked"/>
 			<channel id="device_locked" typeId="device_locked"/>
 			<channel id="temperature" typeId="temperature"/>
-			<channel id="actual_temp" typeId="actual_temp"/>
+			<channel id="actual_temp" typeId="system.indoor-temperature">
+				<label>Current Temperature</label>
+				<description>Current measured temperature.</description>
+			</channel>
```

Comment:
If you change the channel-type, please add update instructions for existing things.

Reply:
They are already part of this PR. Or did I miss one of the changed Things.

Reply:
And I already forgot that I reviewed them. :flushed:

Reply:
By the way: I missed to change the channel types in the https://github.com/openhab/openhab-addons/blob/4fb55d3350dfb3767abf1e1f6b5dfa220be05fc7/bundles/org.openhab.binding.avmfritz/src/main/resources/OH-INF/thing/bridge-types.xml. Will do that.

Do the update instructions work for bridge-types too? I am not able to add tag `<bridge-types>` in the `instructions.xml`.

Reply:
`thing-type` should also work for bridges.